### PR TITLE
Consistency for station pets.

### PR DESCRIPTION
**Pull request by uomo91**
## About The Pull Request

This PR cleans up _some_ of the dumb inconsistencies and oversights with the various station's pets.

- Wags-His-Tail now actually exists in the code as a subtype, and isn't a simple varedit on a regular lizard. He has been placed in Box and Delta, where he was previously missing.
- Cayenne the syndicate carp no longer has AI turned off, they will move around and attack non-Syndicate members.
- Lia the carp made a proper subtye, had their AI turned on with Cayenne's, made neutral, placed in Icebox' HoS office, and given a proper bed instead of Cayenne's bed.
- Sgt. Araneus made a proper subtype and added to Pubbystation's HoS office.
- McGriff the pug made a proper subtype in the code, added to the warden's office on all maps except Pubby where he already was, given a proper bed with his name.
- Paperwork the sloth added to Pubby cargo bay.
- Renault added to Captain's office on Pubby.
- Lamarr has a description now.
- Sgt Araneus, Cayenne, and Lia, and Tom the Mouse are now fun to pet and have a special emote as per https://github.com/tgstation/tgstation/pull/52594
- Tom the mouse added to the permabrigs he was missing from.

## Why It's Good For The Game

🕷️ 🐟 🐕 🦥 🐁
Pets are nice.

## Changelog
:cl:
tweak: Nanotrasen's pet budget has increased, and missing pets have been dispatched to various departments.
/:cl:

### CODE REVIEW

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm [hunk 1]
```diff
@@ -1607,6 +1607,18 @@
...
 /obj/structure/cable,
+/mob/living/simple_animal/hostile/carp/cayenne{
+	desc = "A failed experiment of Nanotrasen to create weaponised carp technology. This less than intimidating carp now serves as an authority figure's pet.";
+	health = 200;
+	icon_dead = "magicarp_dead";
+	icon_gib = "magicarp_gib";
+	icon_living = "magicarp";
+	icon_state = "magicarp";
+	maxHealth = 200;
+	name = "Lia";
```
Comment (ShizCalev) [inline _maps/map_files/IceBoxStation/IceBoxStation.dmm:1618]: Mind making this a subtype too?

File: _maps/map_files/PubbyStation/PubbyStation.dmm [hunk 2]
```diff
@@ -5351,6 +5351,29 @@
...
+	melee_damage_lower = 15;
+	melee_damage_upper = 20;
+	min_oxy = 5;
+	movement_type = 1;
+	name = "Sergeant Araneus";
+	real_name = "Sergeant Araneus";
+	response_help_continuous = "pets";
+	response_help_simple = "pet";
+	turns_per_move = 10
+	},
```
Comment (ShizCalev) [inline _maps/map_files/PubbyStation/PubbyStation.dmm:5376]: Subtype

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: _maps/map_files/IceBoxStation/IceBoxStation.dmm (hunk 1 above)
Comment (uomo91) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm:1618]: Will do.

Comment (uomo91) [reply _maps/map_files/IceBoxStation/IceBoxStation.dmm]: Done.

File: _maps/map_files/PubbyStation/PubbyStation.dmm (hunk 2 above)
Comment (uomo91) [reply _maps/map_files/PubbyStation/PubbyStation.dmm]: Also done.

File: code/modules/mob/living/simple_animal/hostile/carp.dm [hunk 3]
```diff
@@ -233,4 +234,17 @@
...
+	real_name = "Lia"
+	desc = "A failed experiment of Nanotrasen to create weaponised carp technology. This less than intimidating carp now serves as the Head of Security's pet."
+	faction = list("carp")
+	health = 200
+	icon_dead = "magicarp_dead"
+	icon_gib = "magicarp_gib"
+	icon_living = "magicarp"
+	icon_state = "magicarp"
+	maxHealth = 200
+	random_color = 0
```
Comment (ShizCalev) [inline code/modules/mob/living/simple_animal/hostile/carp.dm:248]: FALSE

File: code/modules/mob/living/simple_animal/hostile/retaliate/bat.dm [hunk 4]
```diff
@@ -33,12 +33,33 @@
...
+	icon_gib = "guard_dead"
+	icon_living = "guard"
+	icon_state = "guard"
+	maxHealth = 250
+	max_co2 = 5
+	max_tox = 2
+	melee_damage_lower = 15
+	melee_damage_upper = 20
+	min_oxy = 5
+	movement_type = 1
```
Comment (ShizCalev) [inline code/modules/mob/living/simple_animal/hostile/retaliate/bat.dm:62]: GROUND

**Review by ShizCalev — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/hostile/carp.dm (hunk 3 above)
Comment (uomo91) [reply code/modules/mob/living/simple_animal/hostile/carp.dm]: Done.

File: code/modules/mob/living/simple_animal/hostile/retaliate/bat.dm (hunk 4 above)
Comment (uomo91) [reply code/modules/mob/living/simple_animal/hostile/retaliate/bat.dm]: And also done.

**Pull request merged**